specification_documents/examples/intro_qc2.mzQC

```diff
@@ -33,20 +33,30 @@
                   "value": "LTQ Orbitrap Velos"
                 }
               ]
+            },
+            {
+              "location": "file://tmp/QC2_18052020.mzid",
+              "name": "QC2_18052020",
+              "fileFormat": {
+                "accession": "MS:1002073",
+                "name": "mzIdentML format"
+              }
             }
           ],
           "analysisSoftware": [
             {
               "accession": "MS:1001058",
               "name": "quality estimation by manual validation",
```

Comment:
**question:** Should the analysis software be manual validation and custom tool (currently) or a new CV term for QCloud?